ddtrace/tracer.py

```diff
@@ -140,7 +158,7 @@ def deregister_on_start_span(self, func):
     def debug_logging(self):
         return self.log.isEnabledFor(logging.DEBUG)
 
-    @debug_logging.setter
+    @debug_logging.setter  # type: ignore[misc]
     @deprecated(message="Use logging.setLevel instead", version="1.0.0")
     def debug_logging(self, value):
```

Comment:
can we not have type hinting here?

Reply:
I'm not quite sure if I follow what you mean, could you elaborate?

Reply:
I was just wondering if there would be issues with mypy by doing
~~~ python
def debug_logging(self, value):
    # type: (bool) -> None
~~~
given that I take the hint two lines above is needed to suppress some errors

Reply:
Ahh, I see. This was the mypy error: `error: Decorated property not supported` so not relevant to the type hinting but rather the decorator property itself. As well, I stayed away from type hinting the deprecated methods and properties in this PR, focusing on the main functions instead.

Reply:
Is this resolved? are we able to add type hinting to this function? (we are ignoring the decorator, but we should still be able to add `# type: (bool) -> None` on L163?

Reply:
Yeah, those are orthogonal issues. The ignore is required, and we *could* add typing there.